Detectors/TOF/reconstruction/src/ClustererTask.cxx

```diff
@@ -0,0 +1,95 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file  ClustererTask.cxx
+/// \brief Implementation of the TOF cluster finder task
+
+#include "TOFReconstruction/ClustererTask.h"
+#include "MathUtils/Cartesian3D.h"
+#include "MathUtils/Utils.h"
+#include "SimulationDataFormat/MCCompLabel.h"
+#include "SimulationDataFormat/MCTruthContainer.h"
+
+#include "FairLogger.h"      // for LOG
+#include "FairRootManager.h" // for FairRootManager
+
+ClassImp(o2::tof::ClustererTask)
+
+  using namespace o2::tof;
```

Comment:
please put `;` after `ClassImp(o2::tof::ClustererTask)` as otherwise the formatting is screwed-up.